tests/layer_tests/pytorch_tests/test_polar.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestPolar(PytorchLayerTest):
+    def _prepare_input(self):
+        return (
+            np.array([1.0, 2.0, 3.0], dtype=np.float32),
+            np.array([0.1, 0.2, 0.3], dtype=np.float32)
+        )
+
+    def create_model(self):
+        class PolarModel(torch.nn.Module):
+            def forward(self, abs, angle):
+                real = abs * torch.cos(angle)
+                imag = abs * torch.sin(angle)
+                return torch.stack([real, imag], dim=-1)
```

Comment:
You are not using aten::polar here. Please use it and return the value using `torch.view_as_real`